[CUBRIDMAN-299] [KO] [EN] 오타 등 수정 (#640)
CUBRID/cubrid-manual: en/sql/function/analysis_fn.rst

If it is used as an aggregate function, **CUME_DIST** sorts the data by the order specified in **ORDER BY** clause; then it returns the relative position of a hypothetical row in the rows of aggregate group. At this time, the position is calculated as if a hypothetical row is newly inserted. That is, **CUME_DIST** returns ("cumulated RANK of a hypothetical row" + 1)/("the number of total rows in an aggregate group").

> If it is used as an analytic function, **CUME_DIST** returns the relative position in the value of the group after sorting each row(**ORDER BY**) with each partitioned group(**PARTITION BY**). The relative position is that the number of rows which have values less than or equal to the input argument is divided by the number of total rows within the group(rows grouped by the partition_by_clause or the total rows). That is, it returns (cumulated RANK of a certain row)/(the number or rows within the group). For example, the number of rows which has the RANK 1 is 2, **CUME_DIST** values of the first and the second rows will be "2/10 = 0.2".

Reviewer: ```suggestion
If it is used as an analytic function, **CUME_DIST** returns the relative position of a value in the group after sorting each row (**ORDER BY**) within each partitioned group (**PARTITION BY**). The relative position is calculated by dividing the number of rows with values less than or equal to the input argument by the total number of rows in the group (rows grouped by the partition_by_clause or all rows if no partition is specified). That is, it returns (cumulative rank of a certain row)/(number of rows within the group). For example, if the number of rows with RANK 1 is 2, the **CUME_DIST** values of the first and second rows will be "2/10 = 0.2".
```
